docs/tool-configuration/configuration-terraform.md

```diff
@@ -39,13 +39,13 @@ Run BitOps with the environmental variable `TERRAFORM_APPLY` set to `true` or se
 | var-file       | BITOPS_TF_VAR_FILE   | --var-file            | Terraform Varaible file | `null`  | No       |
 | target         | BITOPS_TF_TARGET     | --target              |                         | `null`  | No       |
 | backend-config |                      | --KEY1=foo --KEY2=bar |                         | `null`  | No       |
+| stack-action | BITOPS_TERRAFORM_COMMAND   | Controls what terraform command to run. e.g. `apply`, `destroy`, etc. | `plan`  | No       |
```

Comment:
This row needs a fix to match the column titles

![image](https://user-images.githubusercontent.com/1533818/203169059-fd6bb206-3f0b-4227-97a9-da879143db16.png)